src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -35,6 +37,11 @@ public User read(@PathVariable Long id) {
     return userRepository.findById(id).orElseThrow(() -> new NotFoundException(ErrorMessages.USER_NOT_FOUND));
   }
 
+  @GetMapping
+  public List<User> readAll() {
+    return (List<User>) userRepository.findAll();
```

Comment:
Why not return an `Iterable`?

Reply:
Because I didn't know the difference with a List object, or that Iterable is returned by this method.